docs/user_guide/models/destvi.rst

```diff
@@ -0,0 +1,206 @@
+
+stLVM
+-----
+
+For the stLVM, we also model the expression counts with a :math:`\mathrm{NegativeBinomial}`. However,
+for spatial data, we assume that each spot :math:`s` has expression :math:`x_s` composed of gene expression
+counts of :math:`C(s)` individual cells. We assume that cell :math:`n` in spot :math:`s` is generated from
+the latent variables :math:`(c_{ns}, \gamma_{ns})` by the following process:
+
+.. math::
+    :nowrap:
+
+    \begin{align}
+        x_{sg} &\sim \mathrm{NegativeBinomial}(l_s\alpha_g\sum_{n=1}^{C(s)}f^g(c_{ns}, \gamma_{ns}), p_g) \tag{3} \\
+    \end{align}
+
+where :math:`l_s` is the library size and :math:`\alpha_g` is a correction term for
+difference in experimental assays. Like the scLVM, :math:`f` is a decoder neural network, and
+:math:`p_g` is the rate parameter for the negative binomial distribution. So, we can model
+the cell type abundances :math:`\beta_{sc}` per spot rather than :math:`C(n)` and the
+corresponding cell types for each cell, we parametrize the sum from :math:`(3)` over cell types.
+
+.. math::
+    :nowrap:
+
+    \begin{align}
+        x_{sg} &\sim \mathrm{NegativeBinomial}(l_s\alpha_g\sum_{c=1}^{C}\beta_{sc}f^g(c, \gamma_s^c), p_g) \tag{4} \\
+    \end{align}
+
+To avoid the latent variable :math:`\gamma_s^c` from incorporating variation attributed to experimental
+assay differences, we assign an empirical prior informed by the scLVM and a corresponding set of
+cells of the same cell type in the scRNA-seq dataset. In literature, the prior is referred to as
+a VampPrior ("variational aggregated mixture of posteriors" prior) [#ref2]_. More can be read on this prior 
+in the DestVI paper.
+
+
+The latent variables for the stLVM, along with their description are summarized in the following table:
+
+.. list-table::
+   :widths: 20 90 15
+   :header-rows: 1
+
+   * - Latent variable
+     - Description
+     - Code variable (if different)
+   * - :math:`\alpha_{g} \in \mathbb{R}^{+}`
+     - Correction term for technological differences.
+     - ``beta``
+   * - :math:`\beta_{sc} \in \mathbb{R}^{+}`
+     - Spot-specific cell type abundance.
+     - ``v_ind``
+   * - :math:`\gamma_s^c \in `
+     - Inverse dispersion for negative binomial. This can be set to be gene/batch specific for example (and would thus be :math:`\theta_{kg}`), by passing ``dispersion="gene-batch"`` during model intialization. Note that ``px_r`` also refers to the underlying real-valued torch parameter that is then exponentiated on every forward pass of the model.
+     - ``gamma``
+   * - :math:`p_g \in \mathbb{R}^{+}`
+     - Inverse dispersion for negative binomial. This can be set to be gene/batch specific for example (and would thus be :math:`\theta_{kg}`), by passing ``dispersion="gene-batch"`` during model intialization. Note that ``px_r`` also refers to the underlying real-valued torch parameter that is then exponentiated on every forward pass of the model.
+     - ``px_o``
+
+
+Inference
+=========
```

Comment:
I'd break this out by sc and st models

Reply:
Also should be q(gamma)?

Reply:
And finally for scLVM I'd refer to the VI tutorial, for stLVM I'd write out the loss explicitly